### ISSUE TITLE
Visual Studio is failing to load 3 FSharp package resources

### ISSUE THREAD

**Issue by davkean** [dotnet/fsharp#10089]
Version: Version 16.8.0 Preview 4.0 [30505.5.main]

Walking through my own ActivityLog.xml trying to investigate some problems, and I see that VS is failing to load 3 strings from the FSharpPackage.

I have no idea the impact of these missing strings, but we should remove the references to them if they aren't needed.

```
641 | Warning | Performance warning: String load failed. Pkg:{871D2A70-12A2-4E42-9440-425DD92A4116} (FSharpPackage) LANG:0409 ID:6000 |   |   | VisualStudio | 2020/09/07 23:37:06.074
-- | -- | -- | -- | -- | -- | --
642 | Warning | Performance warning: String load failed. Pkg:{871D2A70-12A2-4E42-9440-425DD92A4116} (FSharpPackage) LANG:0409 ID:6001 |   |   | VisualStudio | 2020/09/07 23:37:06.075
643 | Warning | Performance warning: String load failed. Pkg:{871D2A70-12A2-4E42-9440-425DD92A4116} (FSharpPackage) LANG:0409 ID:100 |   |   | VisualStudio | 2020/09/07 23:37:06.084
```

**Comment by davkean** [comment]
I see 6000/6001 in https://github.com/dotnet/fsharp/blob/ec5bad3a391357e03ff2286a264f0e4faf7d840d/vsintegration/src/FSharp.ProjectSystem.FSharp/VSPackage.resx, but VS cannot find them because this package's GUID (91A04A73-4F2C-4E7C-AD38-C1A68E7DA05C) doesn't match the above reference. I don't know where 100 lives.

**Comment by cartermp** [comment]
Looks like it's here: https://github.com/dotnet/fsharp/blob/main/vsintegration/src/FSharp.LanguageService/VSPackage.resx#L52

Both are from legacy components. We will load the legacy project system for legacy F# projects. but the one I linked should never get loaded, as it's legacy langauge service bindings that were deprecated in VS 2017. So it appears that at a minimum we have more cleanup to do?

**Comment by davkean** [comment]
Yeah the references to them should be deleted, I couldn't find the reference in a quick search, they will either be an attribute with a # in front it, like this https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeRegistration.cs#L26. Or inside a pkgdef.

**Comment by rober423** [comment]
VS is failing to load 4 strings now from the FSharpPackage for me
